chpl/chpl-service/src/main/java/gov/healthit/chpl/web/controller/ActivityController.java

```diff
@@ -1042,6 +1042,10 @@ private void validateActivityDates(Long startDate, Long endDate) throws Validati
 		Calendar calendarCounter = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
 		Integer maxActivityRangeInDays = Integer.getInteger(env.getProperty("maxActivityRangeInDays"), 60);
 		calendarCounter.setTime(new Date(endDate));
+		if(new Date(startDate).compareTo(calendarCounter.getTime()) > 0){
+			throw new ValidationException("Cannot search an activity with a start date greater than the end date");

```

Comment:
We should try to make errors that end users see as close to English as possible. I'd phrase this error as "Cannot search for activity with the start date after the end date"